hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/SubsystemAdd.java

```diff
@@ -101,13 +96,11 @@ private void createService(final ServiceTarget target, final List<ServiceControl
             svcBuilder.addDependency(binderServiceName);
 
             // install the binder service
-            ServiceController<?> binderController = binderBuilder.install();
-            newControllers.add(binderController);
+            binderBuilder.install();
```

Comment:
yes, this is possible since WildFly 8.x times IIRC.
We also overloaded the performRuntime method and depreacted old one that had list of newControllers in it.
It is all done behind the scenes now.